.reek.yml

```diff
@@ -0,0 +1,9 @@
+detectors:
+  IrresponsibleModule:
+    enabled: false
+
```

Comment:
Do not change this file!

Reply:
I needed to change those files so now they are the same as updated in summer-2019